### PR TITLE
Git FAQs doc - audit

### PR DESCRIPTION
**Pull request by nataliejeremy**
This PR includes general style and grammatical edits.
@bmackinney please review.

### CODE REVIEW

File: source/docs/articles/local/git-faq.md [hunk 1]
```diff
@@ -5,11 +5,17 @@ category:
   - developing
 keywords: git, git commands, conflicts, resolve conflicts, core, drupal, wordpress
 ---
+[Git](https://git-scm.com/) is the version control tool at the heart of the Pantheon workflow. If you're a developer who likes to use [local development](/docs/articles/local), it's a good way to work with the Pantheon platform: develop locally, commit, and push to master to deploy code into your Pantheon Development environment.
 
-## Resolving Conflicts
+## Resolve Conflicts
+
+<p>Conflicts can occur when the upstream you are trying to merge your code with has made alterations to files.</p>
```
Comment (bmackinney) [inline source/docs/articles/local/git-faq.md]: Is there a reason we're using `<p>` tags here? @rachelwhitton do you recall?

**Comment by bmackinney** [comment]
:+1: @rachelwhitton is working on an update to this doc too. please coordinate.

File: source/docs/articles/local/git-faq.md (hunk 1 above)
Comment (rachelwhitton) [inline source/docs/articles/local/git-faq.md]: They were for spacing the following icon, if removed we'd just want to make sure it looks okay when rendered.

**Comment by rachelwhitton** [comment]
The doc I'm updating will be [Applying Upstream Updates](https://pantheon.io/docs/articles/sites/code/applying-upstream-updates/#debug-failed-merges). I'll do that in another PR since it's unrelated.

**Pull request merged**